src/main/java/com/google/step/youtube/DescriptionParser.java

```diff
@@ -11,7 +11,9 @@
  */
 public class DescriptionParser {
 
-    // enum is package private in order to access in tests
+    // package private in order to access in tests
+    static final int MAX_SNIPPET_LENGTH = 200;
```

Comment:
very optional: If you want you can use:
https://guava.dev/releases/19.0/api/docs/com/google/common/annotations/VisibleForTesting.html
in lieu of a comment.

Reply:
Added.